e2e-tests/pages/dashboardPage.ts

```diff
@@ -77,7 +78,8 @@ export class DashboardPage {
         this.dashBoardWithoutMasksEmail = page.locator('//section[starts-with(@class, "profile_no-premium-header")]')
 
         // mask card elements
-        this.maskCard = '//div[starts-with(@class, "MaskCard_card")]'
+        this.maskCard = page.getByRole('button', { name: 'Generate new mask' })
+        this.maskCardString = '//div[starts-with(@class, "MaskCard_card")]'
```

Comment:
Interesting, what does the `starts-with(@class)` do that'd be different than something like this (if I'm reading the code correctly)?
```js
div.MaskCard_card
```

Which maybe I'm not reading it correctly, since I con't find any other instances of "MaskCard_card" outside this ref:
```sh
git grep -in "MaskCard_card" | cat

e2e-tests/pages/dashboardPage.ts:82:        this.maskCardString = '//div[starts-with(@class, "MaskCard_card")]'
```

Reply:
the "starts-with" simply indicates the element's class name starts with that certain value, the full class name for this element is "class="MaskCard_card__oQqFw MaskCard_"is-enabled__nv6Q7 MaskCard_is-not-blocking-promotionals__lel4R MaskCard_is-not-removing-trackers__ev5Is", div.MaskCard_card would not work to locate the element in this instance.

Reply:
there's probably an opportunity for better locator maybe accessibility but thats not within the scope of this pr